meson.build

```diff
@@ -554,6 +555,15 @@ if build_double
   endif
 endif
 
+if build_pythoniface
+  libgalahad_python = library('galahad_python',
+                              sources : libgalahad_python_src,
+                              link_with : libgalahad_c_double,
+                              link_language : 'c',
+                              include_directories: libgalahad_include,
+                              install : true)
+endif
```

Comment:
This creates `/usr/lib/libgalahad_python.so` which... embeds the python interpreter? Seems unlikely.

Maybe you want this instead:
```meson
py = import('python').find_installation()

py.extension_module(
    'foobar',
    'foobar.c',
    dependencies: [py.dependency(), otherdep],
    install: true,
    subdir: 'packagename',
)
```

Using the [python module](https://mesonbuild.com/Python-module.html) has some advantages. On Meson 0.63+ it will automatically handle py.dependency() for you, it runs as a shared module so no need to link to libpython (it isn't recommended by CPython to directly link anyway), it handles the install directory and filename extension, etc.